src/MicrodataParser.php

```diff
@@ -48,34 +51,34 @@ public function toArray() : array
     /**
      * Extracts and converts microdata to object
      *
-     * @return \stdClass
+     * @return stdClass
      */
-    public function toObject() : \stdClass
+    public function toObject() : stdClass
     {
         return $this->extractMicrodata();
     }
 
     /**
      * Extracts and converts microdata to json using \json_encode()
      *
-     * @see \json_encode() to description of parameters and return values
-     *
      * @param int $options
      * @param int $depth
      *
      * @return false|string
+     *@see \json_encode() to description of parameters and return values
```

Comment:
Is there a reason for moving this? If yes, please make this change:
```suggestion
     * @see \json_encode() to description of parameters and return values
```